layouts/partials/home/custom.html

```diff
@@ -65,30 +59,59 @@
 </section>
 
 <script>
-// This bit handles the randomised questions + images
-
 const questions = {{ .Params.questions }}
-const images = {{ .Params.images }}
+const processedImages = window.heroImages || []
 
-// Set a random text + image immediately
 setHeroText()
-setHeroImage()
-
-// Set random text + image every N seconds
-setInterval(setHeroText, 5000)   // 5s
-setInterval(setHeroImage, 13000) // 13s
 
+setInterval(setHeroText, 5000)
+if (processedImages.length > 0) {
```

Comment:
I'm still reading, but this could set up a  race condition if for some reason `window.heroImages` isn't in in time. Maybe instead make `setHeroImage` account for that case?

Reply:
Oh, you already do!
So recommend dropping this `if`